### ISSUE TITLE
🐞 Bug – `a` elements without href causes page reload 

### ISSUE THREAD

**Issue by domenkozar** [elm-land/elm-land#161]
See for more details and a solution 

https://stackoverflow.com/questions/52708345/a-elements-without-href-or-with-an-empty-href-causes-page-reload-when-using-br

**Comment by silky** [comment]
Perhaps relatedly, I think in general links are a bit broken

If you have a page like `/test?a=2`, where you evaluate the querystring in `init`, then you have a link like `/test`; it not only doesn't re-evaluate the init; it doesn't change at all; which feels wrong.

I think there needs to be a way to customise the link behaviour.

**Comment by ryan-haskell** [comment]
Hey @domenkozar, I've changed the generated code to produce this:

```elm
        UrlRequested (Browser.External url) ->
            if String.isEmpty (String.trim url) then
                ( model, Cmd.none )

            else
                ( model
                , Browser.Navigation.load url
                )
```

This bugfix will go live with #173

**Comment by ryan-haskell** [comment]
Good news– I just released [📦 Elm Land v0.20.0](https://github.com/elm-land/elm-land/releases/tag/v0.20.0) which resolves this issue (more details in the release linked above)

Thanks for improving Elm Land! 🌈
